### ISSUE TITLE
V3.0 Inshape asking for updated rezzer?

### ISSUE THREAD

**Issue by britbrit83** [InWorldz/inshape-inworld#1]
Hello I just put the scripts from github inshape 3.0 into the inworldz script files and its telling me:

[20:53] Primitive: I do not have an InShape Runner - you need a newer rezzer
[20:53] Primitive whispers: Please add the route notecard to my contents to get started.

As I put in the route it keeps saying this. Any suggestions?

Thank you,
Brittany

**Comment by adamndyson** [comment]
Hi Brittany.

If you are looking to use the whole kit as it is then your best bet is to pick up the boxed kit at :
http://places.inworldz.com/InWorldz%20InShape/198/234/24

That that has the necessary objects with the scripts already in place. It's still all full-perm so you can modify and edit it if you wish, but it has the benefit of having the scripts on the right places for you the objects for you.

Basically the runner device is the one people sit on to move along the route. You can either use that alone and add your route to it, or you put it in another object with the rezzer script in it then add the notecard to the rezzer object - it passes it to the runner object. The kit also contains instructions and a link to a route making video so it really is an easier place to start.
